### ISSUE TITLE
Getting the ID of a coin?

### ISSUE THREAD

**Issue by Yokomoko** [lzehrung/coinmarketcap#1]
Sorry I might just be misunderstanding how to use the API, but I have this code snippet:

```        public async Task GetPrice(string ticker) {
            ticker = ticker.ToUpper().Trim();

            var listings = await priceClientNew.GetListingsAsync();
            var listingSingle = listings.Data.FirstOrDefault(d => d.Symbol == ticker);

            var tickerResponse = await priceClientNew.GetTickerAsync(listingSingle.Id, "USD");
```

The ID returned from 'listingsSingle' is a long, but the GetTickerAsync method is expecting an Int (and casting to an int doesn't appear to yield any results). 

Is this a bug or am I doing this wrong? (Otherwise, how am I meant to know what ID is for what cryptocurrency?)

**Comment by Yokomoko** [comment]
Wait, casting to an int does appear to work on 1.1.1 version now, but is this still an issue with casting?

**Comment by lzehrung** [comment]
D'oh! That's definitely my bad. I auto generated the response models and (shamefully) hadn't actually tried to use the library myself yet.

1.1 fixed an issue where GetTickerAsync didn't even append the ID to the request URL. First nuget package and I definitely rushed it out the door. Sorry about that.

You should be safe casting it, the largest ID is 2862 right now. I'll publish a new version soon that makes those types match up.

**Comment by lzehrung** [comment]
The cast issue is fixed in 1.2.0 https://www.nuget.org/packages/CoinMarketCap-API/1.2.0

Thanks for the feedback @Yokomoko

**Comment by Yokomoko** [comment]
Thank you :) Good work by the way
